test/blackbox-tests/test-cases/dune

```diff
@@ -0,0 +1,370 @@
+(cram
+ (deps (package dune) (alias test-deps)))
+
+(subdir bisect-ppx
+ (cram
+  ;  disabled for now
+  (enabled_if false)
+  (deps (package dune) (alias test-deps))))
+
+(subdir coq
+ (cram
+  (alias runtest-coq)
+  (deps (package dune) (alias test-deps))))
```

Comment:
I'm wondering if we shouldn't reconsider the idea of inheriting cram settings from the parent. That seems like a simpler way to describe a large test suite. WDYT?

Reply:
Yeah, I got bothered by having to insert all this boilerplate too.

> That seems like a simpler way to describe a large test suite.

In other words, this would mean that
```
(cram (applies_to foo.t) (deps (package dune)))
```
would apply to `./foo.t`, `./*/foo.t`, `./*/*/foo.t`, etc. I think it's fine to do this.

Reply:
Actually, that's not the mental model I had, but that would work too. However we don't really have anything like this in Dune; all the paths and globs are always relative to the current directory.

What I had in mind was that a `cram` stanza with `applies_to` would apply to the whole sub-tree where it appears. In practice, do we need anything more fine-grained than that? It's simple and I feel like it would be enough for Dune for instance, and would probably be enough for other projects as well. WDYT?

Reply:
I've implemented `(applies_to :whole_subtree)` as we discussed.